app/models/hub_edos/student.rb

```diff
@@ -0,0 +1,32 @@
+module HubEdos
+  class Student
+    attr_accessor :terms_in_attendance, :academic_level
+
+    def initialize(uid)
+      @uid = uid
+      if uid.present?
+        @student_hash = HubEdos::V2::Student.new(user_id: uid).get
+      end
+    end
+
+    def get_max_terms_in_attendance
+      if statuses = @student_hash[:feed]['academicStatuses']
+        return statuses.collect {|s| s['termsInAttendance']}.sort.last
+      end
+    end
+
+    def get_student_academic_level
+      current_registration = get_current_registration
+      type_code = current_registration['academicCareer']['code'] == 'LAW' ? 'EOT' : 'BOT'
+      level = current_registration['academicLevels'].find {|al| al['type']['code'] == type_code }
+      level['level']['description']
+    end
+
```

Comment:
Is `get_current_registration` part of the public API for this class? If not, consider making this a private method

Reply:
That led me to some complicated testing issue, but I just decided to not include a test that ensures that the private method memoizes it's output